templates/dashboard/_grid_information.html.twig

```diff
@@ -0,0 +1,11 @@
+{% trans_default_domain 'dashboard' %}
+
+{% for column in grid_information %}
+    <div class="col-md-6 col-lg-3 mb-5 mb-md-0">
+        <h2 class="h3"><i class="fas fa-list mr-2"></i>{{ column.grid_label|trans }}<span class="badge badge-dark ml-2">{{ column.hearing_data.count + column.new_party_submission_data.count + column.agenda_data.count + column.exceeded_deadline_data.count }}</span></h2>
+        <a type="button" class="btn btn-primary btn-lg btn-block" href="{{ column.hearing_data.url }}">{% trans %}Hearing in progress{% endtrans %}<span class="badge badge-light ml-2">{{ column.hearing_data.count }}</span></a>
```

Comment:
The `type` attribute an the `a` element cannot be used like this (cf. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-type) and should be removed here.

Reply:
Removed `type` attribute.